src/server/script_mgr.h

```diff
@@ -31,8 +31,7 @@ class ScriptMgr {
   };
 
   struct ScriptData : public ScriptParams {
-    std::string body;       // script source code present in lua interpreter
-    std::string orig_body;  // original code, before removing header and adding async
```

Comment:
it still can be different for lua_auto_async, but yeah, we never actually needed it for sure